lib/configuration.yaml

```diff
@@ -14939,19 +14939,16 @@ testnet_wallet_win64: &testnet_wallet_win64
   <<: *testnet_wallet
   update:
     <<: *testnet_wallet_update
```

Comment:
```
  update:
    <<: *testnet_wallet_update
```
these 2 lines can be removed from each block, since your no longer changing anything in update